src/test/java/core/basesyntax/RegistratonServiceImplTest.java

```diff
@@ -0,0 +1,108 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistratonServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.PEOPLE.add(new User("user01", "password123", 25));
+    }
+
+    @Test
+    void invalidLogin_NotOk() {
```

Comment:
```suggestion
    void register_invalidLogin_NotOk() {
```
![image](https://github.com/mate-academy/jv-registration-validation/assets/20326858/fb104bf5-2e00-4c49-a0ed-892b8e3fb93a)